.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
I'd still rather not clutter the project with 3rd party files unrelated to the project. I just went through normalizing the files and it seems like `keymaps/code-links.cson` is the only thing that had CRLF line endings.

I use https://atom.io/packages/project-manager and it does this and more.